js/plugins/code/base-text-element.js

```
@@ -1671,46 +1903,107 @@ class BaseTextElement extends HTMLElement {
                         }
+                        skipChildren = true;
                         break;
                     case 'p':
                         if (node.textContent.trim()) {
-                            element = { elementName: 'text-element', value: node.textContent.trim() };
+                            element = { elementName: 'text-element', value: makeLinksClickable(node.innerHTML.trim()) };
+                        }
+                        skipChildren = true;
+                        break;
+                    case 'table':
+                        const headers = [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)